apps/users/templates/user/profile.html

```diff
@@ -15,37 +15,37 @@
     <div class="col-md-2">
         <ul class="nav nav-pills flex-column">
             <li class="nav-item d-grid">
-                <button class="nav-link active" id="profile-pill" data-bs-toggle="pill" data-bs-target="#profile" role="tab" aria-current="page">Profile</a>
+                <button class="nav-link {% if active_tab == 'profile' %}active{% endif %}" id="profile-pill" data-bs-toggle="pill" data-bs-target="#profile" role="tab" aria-current="page">Profile</a>
             </li>
             <li class="nav-item d-grid">
-                <button class="nav-link" id="password-pill" data-bs-toggle="pill" data-bs-target="#password" role="tab">Password</a>
+                <button class="nav-link {% if active_tab == 'password' %}active{% endif %}" id="password-pill" data-bs-toggle="pill" data-bs-target="#password" role="tab">Password</a>
             </li>
             <li class="nav-item d-grid">
-                <button class="nav-link" id="preferences-pill" data-bs-toggle="pill" data-bs-target="#preferences" role="tab">Preferences</a>
+                <button class="nav-link {% if active_tab == 'preferences' %}active{% endif %}" id="preferences-pill" data-bs-toggle="pill" data-bs-target="#preferences" role="tab">Preferences</a>
```

Comment:
I thought that bootstrap inherently already had functionality for this by adding e.g. #preferences to the url, could you double check that? Cause I think that would be a lot nicer than this. Furthermore, what will happen if no active_tab is passed along?

Reply:
I tried to find something nicer when I first wrote the code and I double checked now but I can't seem to find any other way to do this except for using jQuery which I don't think would be better so I hope I'm not missing something. 

As for no active_tab being passed along, it has a default value of "profile" so that shouldn't be a problem.